app/controllers/grant_permissions_controller.rb

```diff
@@ -32,7 +28,7 @@ def create
     @grant_permission = GrantPermission.new(grant_permission_params)
     respond_to do |format|
       if @grant_permission.save
-        flash[:notice] = full_name(@grant_permission.user) + ' was granted \'' + @grant_permission.role + '\' permissions for this grant.'
+        flash[:notice] = helpers.full_name(@grant_permission.user) + ' was granted \'' + @grant_permission.role + '\' permissions for this grant.'
```

Comment:
Maybe the 'full_name' belongs in the User model.

Reply:
We have a ticket for just this -- #159